hydromt_sfincs/sfincs.py

```diff
@@ -1316,6 +1317,49 @@ def setup_observation_points(
         self.set_geoms(gdf_obs, name)
         self.set_config(f"{name}file", f"sfincs.{name}")
 
+    def setup_observation_lines(
+        self,
+        locations: Union[str, Path, gpd.GeoDataFrame],
+        merge: bool = True,
+        **kwargs,
+    ):
+        """Setup model observation lines (cross-sections) to monitor discharges.
+
+        Adds model layers:
+
+        * **crs** geom: observation lines (cross-sections)
+
+        Parameters
+        ---------
+        locations: str, Path, gpd.GeoDataFrame, optional
+            Path, data source name, or geopandas object for observation lines (cross-sections).
+        merge: bool, optional
+            If True, merge the new observation lines with the existing ones. By default True.
+        """
+        name = self._GEOMS["observation_lines"]
+
+        # FIXME ensure the catalog is loaded before adding any new entries
+        self.data_catalog.sources
```

Comment:
Why is this needed?

Reply:
Good question, copied this from the setup_observation_points. Can this be related to the fact that self.data_catalog normally keeps tracks of all the used data_sources (something which seems to be disappeared along the way?) and therefore we needed to ensure that the data_catalog was loaded?

Maybe this still originates from the first codes and the problem has been fixed in the meanwhile?